### PR TITLE
removes record side navigation

### LINKED ISSUE #28: Remove sidebar navigation menu
per @jvine

### PR DESCRIPTION
**Pull request by mejackreed**
Closes #28

### CODE REVIEW

**Comment by cbeer** [comment]
Remove https://github.com/sul-dlss/purl/blob/sw-design/vendor/assets/javascripts/jquery-scrollspy.js too?

File: app/views/purl/_mods_metadata_sections.html.erb [hunk 1]
```diff
@@ -33,8 +33,6 @@
   </div>
 <% end %>
 
-<%= render partial: "record_side_nav", locals: locals %>
```
Comment (cbeer) [inline app/views/purl/_mods_metadata_sections.html.erb]: You can also remove the `locals` assignment at the top of this file.

File: app/views/purl/_mods_metadata_sections.html.erb [hunk 2]
```diff
@@ -33,8 +33,6 @@
...
-
 <% if contributors.present? %>
   <div class="section" id="<%= t('record_side_nav.contributors.id') %>" data-side-nav-class="<%= t('record_side_nav.contributors.id') %>">
```
Comment (cbeer) [inline app/views/purl/_mods_metadata_sections.html.erb]: And maybe the `data-side-nav-class` attributes?

Comment (mejackreed) [inline app/views/purl/_mods_metadata_sections.html.erb]: :+1:

File: app/views/purl/_mods_metadata_sections.html.erb (hunk 1 above)
Comment (mejackreed) [inline app/views/purl/_mods_metadata_sections.html.erb]: :+1:

**Comment by mejackreed** [comment]
:construction:  Removed two jQuery Plugins, a stylesheet, and a partial... My work here is done

**Pull request merged**